### PR TITLE
Updated cuationary note about screens using Astro 4

### PR DESCRIPTION
**Pull request by dmcalester**
Added a banner to service specific designs noting screens should not be used for color/font. Adding links after draft-pr generates preview

### CODE REVIEW

**Comment by carmenparks** [comment]
@dmcalester Awkward/repetitive working of this sentence:
"All new projects should use Astro 5 colors and fonts should be used for compliant Astro applications."

Suggest removing words in bold: All new projects should use Astro 5 colors and fonts **should be used** for compliant Astro applications.

Also needs space after the period at the end of this sentence.

**Review by carmenparks — CHANGES_REQUESTED** [review]
Left comment re: repetitive wording and needing space after end of sentence.

File: _content/grm-service-ux-design/about-the-grm-designs.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ title: About the GRM Designs
 
 # Ground Resource Management (GRM)
 :::note
-The images depicted on this page use the color palette and fonts from Astro 4. As of X/XX/XXXX Astro 5 colors and fonts should be used for compliant Astro applications.Refer to this section for general user experience guidance only, *not* visual design guidance
+The images depicted on this page use the color palette and fonts from Astro 4. All new projects should use Astro 5 colors and fonts should be used for compliant Astro applications.Refer to this section for general user experience guidance only, *not* visual design guidance
```
Comment (amalyah) [inline _content/grm-service-ux-design/about-the-grm-designs.md:11]: Same as Carmen's comment for this line, but the sentence at the end of the paragraph needs a period too. Needs to be updated on all instances.

**Review by amalyah — CHANGES_REQUESTED** [review]

**Comment by amalyah** [comment]
Which Jira ticket does this PR link to @dmcalester?

**Comment by Design-Kat** [comment]
"All new projects should use Astro 5 colors and fonts should be used for compliant Astro applications."
This sentence feels incomplete and the meaning isn't quite clear. I'm not sure if you're talking about the same "new projects" on the first and second half or if one is for new and one is for old? Suggest a bit of a tweak to how it's written. If they're the same "All new projects should Use Astro 5 colors and font to be considered a compliant Astro application." or if referring to New vs Established applications consider rewording to something like, "All new projects should Use Astro 5. Currently compliant Astro Applications should Use Astro 5 colors and fonts."

File: _content/grm-service-ux-design/about-the-grm-designs.md (hunk 1 above)
Comment (Design-Kat) [reply _content/grm-service-ux-design/about-the-grm-designs.md:11]: "All new projects should use Astro 5 colors and fonts should be used for compliant Astro applications."
This sentence feels incomplete and the meaning isn't quite clear. I'm not sure if you're talking about the same "new projects" on the first and second half or if one is for new and one is for old? Suggest a bit of a tweak to how it's written. If they're the same "All new projects should Use Astro 5 colors and font to be considered a compliant Astro application." or if referring to New vs Established applications consider rewording to something like, "All new projects should Use Astro 5. Currently compliant Astro Applications should Use Astro 5 colors and fonts."

**Review by Design-Kat — CHANGES_REQUESTED** [review]
See comment on about-the-grm-designs.

**Review by Design-Kat — CHANGES_REQUESTED** [review]
Only change I see is that the 's' on Applications should be removed, or the 'an' removed to make the plural applications make sense - other than that it looks great!

**Review by amalyah — APPROVED** [review]

**Comment by amalyah** [comment]
Jira ticket: https://rocketcom.atlassian.net/browse/ASTRO-970

**Review by Design-Kat — APPROVED** [review]

**Pull request merged**